initdb.d/migrate.sql

```diff
@@ -11,13 +11,13 @@ CREATE TABLE users (
     image TEXT,                       -- image: 画像のパス・base64
     github_url VARCHAR(255),             --link:Githubのリンク
     x_url VARCHAR(255),             --link:Xのリンク
-    portfolio_url VARCHAR(255)        --link:Githubのリンク
+    portfolie_url VARCHAR(255)        --link:Githubのリンク
 );
 
-INSERT INTO users (name, displayName, bio, image, email, password, github_url, x_url, portfolio_url)
+INSERT INTO users (name, displayName, bio, image, email, password, github_url, x_url, portfolie_url)
 VALUES
-('山田太郎', 'Taro Yamada', '日本のエンジニア。プログラミングが得意です。', 'path/to/image1.jpg', 'taro.yamada@example.com', crypt('password123', gen_salt('bf')),'' ,'' ,''),
-('鈴木花子', 'Hanako Suzuki', 'ウェブデザイナー。美しいデザインを作成します。', 'path/to/image2.jpg', 'hanako.suzuki@example.com', crypt('mypassword', gen_salt('bf')), '' ,'' ,''),
+('山田太郎', 'Taro Yamada', '日本のエンジニア。プログラミングが得意です。', 'https://irasutoya.jp/wp-content/uploads/2021/01/kawa-pikach-no-irasuto-t-mei.png', 'taro.yamada@example.com', crypt('password123', gen_salt('bf')),'github.com' ,'x.com' ,'taro.yamada@example.com'),
```

Comment:
```suggestion
('山田太郎', 'Taro Yamada', '日本のエンジニア。プログラミングが得意です。', 'https://irasutoya.jp/wp-content/uploads/2021/01/kawa-pikach-no-irasuto-t-mei.png', 'taro.yamada@example.com', crypt('password123', gen_salt('bf')),'https://github.com' ,'https://x.com' ,''),
```

外部サイトはちゃんと`https://`から始める
メアドじゃなくてサイトのリンクを入れるところだから何もなければ空文字でいいと思う。

Reply:
`佐藤一郎`さんのアイコン画像も別のに入れ替えて欲しい